package.json

```diff
@@ -8,7 +8,9 @@
     "start": "node app.js",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
+    "swagger": "node ./swagger.js",
+    "start-gendoc": "node ./src/app.js"
```

Comment:
This just starts the app right, do we need this `start-gendoc` script?

Reply:
We can delete it actually.

Reply:
I have deleted this line , it should be there actually
`  "scripts": {
    "dev": "nodemon src/app.js",
    "start": "node app.js",
    "lint": "eslint .",
    "lint:fix": "eslint . --fix",
    "format": "prettier --write ."
  },`

Reply:
Done commit b3b7a9f8646209f3ee622108ca54cf7510a28261